### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Booklight12** [AndroidIDEOfficial/AndroidIDE#1007]
### What happened?

i do not know

### What's the expected behavior?

modify background

### What version of AndroidIDE you're using?

v2.4.0 (debug builds)

### Relevant log output

```shell
AndroidIDE crash report
Manufacturer: Xiaomi
Device: 22041216C
ABI: arm64-v8a
SDK version: 33
App version: v2.4.1-beta (241)

 Stacktrace: 
kotlinx.coroutines.CompletionHandlerException: <selector> item does not define android:drawable
	at com.itsaky.androidide.inflater.drawable.StateListParser.parseDrawable(StateListParser.kt:210)
	at com.itsaky.androidide.inflater.drawable.IDrawableParser.parse(IDrawableParser.kt:64)
	at com.itsaky.androidide.inflater.internal.utils.BasicParseUtilsKt$parseDrawable$drawableResolver$1.invoke(basicParseUtils.kt:50)
	at com.itsaky.androidide.inflater.internal.utils.BasicParseUtilsKt.resolveResourceReference(basicParseUtils.kt:11)
	at com.itsaky.androidide.inflater.internal.utils.BasicParseUtilsKt.parseReference(basicParseUtils.kt:336)
	at com.itsaky.androidide.inflater.AbstractParser.parseDrawable(AbstractParser.kt:10)
	at com.itsaky.androidide.inflater.AbstractParser.parseDrawable$default(AbstractParser.kt:15)
	at com.itsaky.androidide.inflater.internal.adapters.ViewAdapter$createAttrHandlers$2.invoke(ViewAdapter.kt:17)
	at com.itsaky.androidide.inflater.IViewAdapter.applyInternal(IViewAdapter.kt:20)
	at com.itsaky.androidide.inflater.IViewAdapter$apply$1.invoke(IViewAdapter.kt:10)
	at com.itsaky.androidide.inflater.IViewAdapter.doApply(IViewAdapter.kt:62)
	at com.itsaky.androidide.inflater.IViewAdapter.apply(IViewAdapter.kt:16)
	at com.itsaky.androidide.inflater.internal.ViewImpl.applyAttribute(ViewImpl.kt:40)
	at com.itsaky.androidide.uidesigner.fragments.AttrValueEditorFragment$showAttrInfo$1$2.invoke(AttrValueEditorFragment.kt:65)
	at com.itsaky.androidide.uidesigner.utils.ValueCompletionProvider$afterTextChanged$1$$ExternalSyntheticLambda0.run(R8$$SyntheticClass:17)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:210)
	at android.os.Looper.loop(Looper.java:299)
	at android.app.ActivityThread.main(ActivityThread.java:8136)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1028)
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Booklight12** [comment]
Delete background in the designer

**Comment by itsaky** [comment]
This was fixed in 111a15d. However, the commit is not part of any release yet. The fix will be available in next release.
